src/mmw/js/src/analyze/views.js

```diff
@@ -104,6 +116,16 @@ var AnalyzeWindow = Marionette.LayoutView.extend({
     }
 });
 
+var LoadingView = Marionette.ItemView.extend({
+    template: loadingTmpl,
+    className: 'analyze-message-region'
+});
+
+var ErrorView = Marionette.ItemView.extend({
+    template: errorTmpl,
+    className: 'analyze-message-region'
+});
+
```

Comment:
Consider consolidating these into one view that expects a model with `message` and `icon-class` attribute.